runtime/lib/app-runtime.js

```diff
@@ -457,6 +457,54 @@ AppRuntime.prototype.onVoiceCommand = function (asr, nlp, action, options) {
     })
 }
 
+/**
+ * handle mqtt 'event' message
+ * @param {string} message message string received from mqtt
+ */
+AppRuntime.prototype.onAppEvent = function (message) {
+  var data = {}
+  try {
+    data = JSON.parse(message)
+  } catch (error) {
+    data = {}
+    logger.debug('parse mqtt event message error: message -> ', message)
+    return
+  }
+  var skillId = data.appId || data.domain
```

Comment:
`data.domain` is specific to mqtt forward. Thus no need to add these conditions here.

Reply:
Removed. Thx!